src/storage/src/redis_pkhashes.cc

```
@@ -0,0 +1,1279 @@
+  std::string value;
+  return PKHGet(key, field, &value);
+}
+
+Status Redis::PKHDel(const Slice& key, const std::vector<std::string>& fields, int32_t* ret) {
+  uint32_t statistic = 0;
+  std::vector<std::string> filtered_fields;
+  std::unordered_set<std::string> field_set;
+  for (const auto& iter : fields) {
+    const std::string& field = iter;
+    if (field_set.find(field) == field_set.end()) {
+      field_set.insert(field);
```

cppcheck

[performance] Searching before insertion is not necessary.

(stlFindInsert)